doc/manual/manual.tex

```diff
@@ -9474,6 +9474,211 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
 
 The parameter file and initial temperature model for this benchmark can be found at \url{benchmarks/yamauchi_takei_2016_anelasticity/yamauchi_takei_2016_anelasticity.prm} and \url{benchmarks/yamauchi_takei_2016_anelasticity/anelasticity_temperature.cc}. Code to recreate  Fig.~\ref{fig:anelasticity}  is provided in \url{benchmarks/yamauchi_takei_2016_anelasticity/plot_output}.
 
+\subsubsection{Thin shell gravity benchmark}
+\label{sec:benchmark-thin-shell-gravity}
+
+\textit{This section was contributed by Cedric Thieulot, Bart Root and Paul Bremmer.}
+
+The domain is a spherical shell of 10~\si{\km} radius centered at a depth $D$, i.e. the inner radius is $R_i=6371-D-5~\si{\km}$ and the outer radius $R_o=6371-D+5~ \si{\km}$. It is filled with a fluid of constant density $\rho_0=3300~\si{\kg\per\cubic\metre}$ with total mass $M=\frac43 \pi (R_o^3-R_i^3)$.
+
+\noindent Both the Stokes and energy equations solve are bypassed:
+
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/gravity_thin_shell/doc/thinshell_b.prm.out}
+
+We make use of the Custom mesh subdivision option to generate a mesh where a single cell is used in the radial direction (parameterized with the number of 'slices') while 6 blocks (default value) of $2^5\times 2^5$ cells are used in the lateral direction. This gives a total of 6,144 mesh cells. For $D=100~\si{\km}$ the parameterization of the mesh is then as follows:
+
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/gravity_thin_shell/doc/thinshell_a.prm.out}
+
+We wish to compute the gravity potential $U$ and the gravity vector ${\mathbf g}$ at a height of $250~\si{\km}$ above the surface of the Earth (i.e. the satellite is flying at a radius of $R_s=6371+250=6621~\si{\km}$) on a regular longitude-latitude grid spanning the whole Earth, with a $2\si{\degree}\times 2\si{\degree}$ resolution:
+
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/gravity_thin_shell/doc/thinshell_c.prm.out}
+
+The Gauss-Legendre Quadrature (GLQ) algorithm is central to the ASPECT code as it is used to compute the elemental integrals stemming from the discretisation of the weak form of the PDEs. Logically, the gravity postprocessor is also based on the GLQ since the potential $U$ at any location in space is given by the integral 
```

Comment:
```suggestion
The Gauss-Legendre Quadrature (GLQ) algorithm is central to the ASPECT code as it is used to compute the elemental integrals stemming from the discretization of the weak form of the PDEs. Logically, the gravity postprocessor is also based on the GLQ since the potential $U$ at any location in space is given by the integral 
```

Reply:
```suggestion
The Gauss-Legendre Quadrature (GLQ) algorithm is central to the \aspect{} code as it is used to compute the elemental integrals stemming from the discretisation of the weak form of the PDEs. Logically, the gravity postprocessor is also based on the GLQ since the potential $U$ at any location in space is given by the integral 
```

Reply:
@bobmyhill That must have hurt to correct @cedrict to use AE spelling :-)

Reply:
exactly my thought :)